packages/block-library/src/post-author/editor.scss

```diff
@@ -0,0 +1,27 @@
+.block-editor__container .wp-block-post-author {
+	display: flex;
+	flex-wrap: wrap;
+	line-height: 1.5;
+}
+
+.wp-block-post-author__byline {
+	width: 100%;
+}
+
+.block-editor__container .wp-block-post-author__avatar {
+	flex-grow: 1;
```

Comment:
If you remove this, and also where it's being set in `/post-author/style.scss`, then you can set `.wp-block-post-author__content` below to `flex-grow: 1`(and same for the `style.scss` one). And we can avoid a z-index wars type situation here 🙀

Reply:
done!